chart/templates/_helpers.yaml

```diff
@@ -15,6 +15,32 @@
 # specific language governing permissions and limitations
 # under the License.
 
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "airflow.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "airflow.fullname" -}}
+{{- if .Values.fullnameOverride }}
```

Comment:
This is not in the chart README.

Reply:
Why do we need two separate override values?

Reply:
I'll get them added to the README 👍 

When you issue a [helm create](https://helm.sh/docs/helm/helm_create/) command, the additions above (and [others](https://helm.sh/docs/helm/helm_create/)) are added to the `_helpers.yaml` file to help follow the best practices set forward by the helm team. Any explanation on the need for the two separate options can be found [here](https://stackoverflow.com/a/63839389)